docs/DeveloperGuide.adoc

```diff
@@ -1507,6 +1520,197 @@ Use case ends.
 +
 Use case resumes at step 1.
 
+[discrete]
+=== Use case: Add a note to a module
+
+*Pre-condition*
+
+Module must exist in data file.
+
+*MSS*
+
+1.  Teacher wants to add a note to a module.
+2.  System displays message of successful addition of a note to the module.
+
```

Comment:
Add `Use case ends.` after every MSS, as per the AB4 guide

Reply:
I concur.

Reply:
Changed as requested.

Reply:
Looks good. Well done Cruz.